Frontend/src/api/user/post/get_user_info.ts

```diff
@@ -0,0 +1,28 @@
+import createClient from 'openapi-fetch'
+import { BackendError } from '../../helper/backend_fetch'
+import { components, paths } from '../../schema'
+
+const { POST } = createClient<paths>({
+  // TODO: Change this once we are fully migrated from backend fetch
```

Comment:
Why is this needed right now?

Reply:
There's still a couple of methods that use backend fetch, because they haven't been added to the swagger definition. Definitely something we should do very soon!
Backendfetch requires the API URL to be the full URL (+ the /api/v1), while createClient doesn't